nxdrive/options.py

```diff
@@ -241,6 +251,14 @@ class MetaOptions(type):
         "use_analytics": (False, "default"),
     }
 
+    # add dynamic options from Features
```

Comment:
```suggestion
    # Add dynamic options from Features
```

Great idea! :) :muscle: